source/style-guide/style/prepositions.txt

```diff
@@ -4,8 +4,6 @@
 Prepositions
 ============
 
-.. include:: /style-guide/includes/page-needs-update.rst
-
 Use prepositions as needed, even at the ends of sentences.
 
 There's no rule against placing a preposition at the end of a sentence.
```

Comment:
Open Q: What is our stance on prepositions at the end of sentences? I've been editing them out, but I don't have strong feelings on this.

Reply:
Same here, but CMOS is ok with it: https://www.chicagomanualofstyle.org/book/ed17/part2/ch05/psec180.html